releases/release-1.31/release-notes/maps/pr-125782-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 125782
+releasenote:
+  text: |-
+    Fixed the documentation for the default value of the procMount entry in the pod securityContext.
+    The documentation was previously using the name of the internal variable 'DefaultProcMount' rather than the actual value 'Default'.
```

Comment:
```suggestion
    The documentation was previously using the name of the internal variable `DefaultProcMount`, rather than the actual value, "Default".
```